src/constants/index.tsx

```diff
@@ -202,6 +202,15 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
     color: '#010101',
     primary: true
   },
+  TALLY: {
+    connector: injected,
+    name: 'Tally',
+    iconName: 'tally.svg',
+    description: 'Connect using Tally Wallet',
```

Comment:
Can you end this description with a period please?

Reply:
No problem, done.